cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -489,6 +509,7 @@ instance ( ConvertRawHash blk
          , InspectLedger blk)
       => HasTextFormatter (ChainDB.TraceEvent blk) where
     formatText tev _obj = case tev of
+      ChainDB.TraceLastShutdownUnclean -> "TraceLastShutdownUnclean"
```

Comment:
This should have a descriptive message. The idea was something like:
```
INFO: ChainDB is not clean. Validating all immutable chunks.
```

Which also implies that this tracing should be on Info, not on Debug.

Reply:
More context: https://github.com/IntersectMBO/ouroboros-consensus/issues/1195